src/FingerprintableEntity.js

```diff
@@ -0,0 +1,45 @@
+( function( wb, $ ) {
+	'use strict';
+
+var PARENT = wb.datamodel.Entity;
+
+/**
+* Abstract FingerprintableEntity class featuring an id and a fingerprint.
+* @class wikibase.datamodel.FingerprintableEntity
+* @abstract
+* @since 0.3
```

Comment:
this is not really true I believe, as the current version is 4.0.0

Reply:
good catch